extensions/amp-install-serviceworker/0.1/amp-install-serviceworker.js

```diff
@@ -92,27 +92,20 @@ export class AmpInstallServiceWorker extends AMP.BaseElement {
     }
   }
 
+  /** @override  */
+  getPriority() {
+    return 3;
+  }
```

Comment:
It seems priority only affect `#layoutCallback`, but we're installing the iframe/SW in `#buildCallback`.

Reply:
Nice catch. Perhaps we should rename `getPriority` to `getLayoutPriority`.

Reply:
Sure.

Reply:
@jridgewell PTAL, moved insertion of the iframe into layout phase. I also changed the description of #13555 to mention renaming the getPriority method as @choumx suggsted